.github/actions/install-build/action.yml

```
@@ -26,4 +26,4 @@ runs:
     - name: Run build
       shell: bash
       working-directory: ${{ inputs.working_directory }}
-      run: pnpm -- turbo run build --filter=woocommerce
+      run: pnpm -- turbo run turbo:build --color --filter=woocommerce
```

```suggestion
      run: pnpm run build --filter="${{ inputs.working_directory }}"
```
Since this action has a working directory, how about we use that to decide what build to execute? We can even use the package's build script to just operate the way PNPM wants to.

The caveat is that the build output of `./` is a little bit strange. It runs the `turbo run build` for every package. It uses the cache so it's not the worst thing ever, but, it's a little odd looking.

I'll leave this one up to you!

---

I'm not strongly opinionated on this one. I can see it is kinda more flexible to use the working directory so lets try that